Package.swift

```diff
@@ -21,8 +21,10 @@ let package = Package(
             name: "BonMot",
             dependencies: [],
             path: "Sources",
-            exclude: ["Info.plist"]
-        ),
+            exclude: ["Info.plist"],
+            resources: [
+                .copy("PrivacyInfo.xcprivacy"),
```

Comment:
Why `copy` instead of `process`?

Reply:
As we are not applying a platform-specific rule to the resource we need copy of it.